book/website/collaboration/academic-industry/case-studies-and-best-practices/practitioner's-hub-case-study.md

```diff
@@ -0,0 +1,24 @@
+# Case Study: Turing Way Practitioner's Hub 
```

Comment:
You should add one/two/three sections to the file describing what went well (best practices), challenges, and lessons learnt from Practitioners' Hub. This way we can start building the best practices and lessons learnt library over time too